### PR TITLE
Fix docusaurus example to work with beta release

### LINKED ISSUE #189: docusaurus example: update dependencies/configs?
Is it possible the example can be updated to work with docusaurus v2 beta?

### PR DESCRIPTION
**Pull request by chris-stetter**
Since some beta version new files and folders with a prefix underscore will be excluded by docusarus (https://github.com/facebook/docusaurus/issues/6044). The example generates underscore files from the corresponding Python files though. I have updated doscusaurus and the documentation.

### CODE REVIEW

File: .changelog/_unreleased.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+changes:
+- type: fix|docs
+  component: docusaurus
+  description: 'Update `plugin-content-docs` config for docusarus@beta to not exclude files and folders with underscores. (@chrisstetter)'
+  fixes: [ '#189' ]
```
Comment (NiklasRosenstein) [inline .changelog/_unreleased.yml:5]: Why does this fix #189 ?

File: .changelog/_unreleased.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+changes:
+- type: fix|docs
```
Comment (NiklasRosenstein) [inline .changelog/_unreleased.yml]: This needs to be just `fix` or `docs` 🙂 

```suggestion
- type: docs
```

**Review by NiklasRosenstein — CHANGES_REQUESTED** [review]

**Comment by NiklasRosenstein** [comment]
Thanks @chrisstetter , happy to merge after the little fix I requested. 👍

File: .changelog/_unreleased.yml (hunk 2 above)
Comment (chris-stetter) [reply .changelog/_unreleased.yml]: This has been fixed now. Happy to contribute!

**Comment by lucacorbucci** [comment]
Hi @chrisstetter I am trying to run the example with docusaurus but I am having some problems. 
Do you know if the latest version of docusaurus has broken anything?

**Comment by chris-stetter** [comment]
I have not tried the latest version yet, so unfortunately I do not know. It is on my agenda but will take some time to be honest.

**Comment by lucacorbucci** [comment]
Thank you for your reply @chrisstetter

**Pull request merged**